### ISSUE TITLE
Provide a way to globally notify of runner install changes

### ISSUE THREAD

**Issue by strycore** [lutris/lutris#295]
After some recent changes made to the side bar, I noticed the console being spammed with "which" calls every 5 seconds. It appears that the client is constantly polling for the install state of every runner and I am not OK with that in the same way that I'm not OK with polling appmanifest files as I stated in issue #212. 

I tried finding a solution where every runner installation would trigger a "runners-changed" signal but couldn't find a way to connect to a signal globally without having a reference to the widget that emitted this signal.

Until this is implemented in some other way, the sidebar won't be updated with newly installed (or removed) runners. 
As a rule of thumb and outside the process monitoring of games, the lutris client shouldn't act like this: https://www.youtube.com/watch?v=18AzodTPG5U

**Comment by Xodetaetl** [comment]
> Until this is implemented in some other way, the sidebar won't be updated with newly installed (or removed) runners.

AFAIK, the issue and its console spam side-effect are an annoyance to us, not to average users. So, as long as the feature is working as intended from user standpoint, it's much better to have it than not. 
Besides, enduring the console spam encourages us to find a solution more quickly. 8)

**Comment by strycore** [comment]
Nevermind, I found a solution (asking on StackOverflow and answering my own question 20 minutes later ^^). Now, I've tested this and the runner list doesn't update with newly updated runners but the complete list is available in the sidebar (looking at the results of self.used_runners = runners.get_installed() in update()) which makes me believe that the initial code wasn't working to begin with (I don't know, I've never tested it :p).
Anyway that's a different issue. Oh, and what about the removed runners? Well, when there's a way to uninstall runners from the RunnerDialog, it will be easy to do the exact same thing (but the signal should be renamed to runners-changed instead of runner-installed)
